diff --git a/node_modules/@agoric/solo/src/start.js b/node_modules/@agoric/solo/src/start.js
index e9c6684..42585e0 100644
--- a/node_modules/@agoric/solo/src/start.js
+++ b/node_modules/@agoric/solo/src/start.js
@@ -483,8 +483,11 @@ const start = async (basedir, argv) => {
   const { 'agoric-wallet': { htmlBasedir = 'ui/build', deploy = [] } = {} } =
     JSON.parse(fs.readFileSync(pjs, 'utf-8'));
 
+  const htmlBasePath = String(htmlBasedir).replace(/^\.\.\/\.\.\/node_modules\//, '');
+
   const agWallet = path.dirname(pjs);
-  const agWalletHtml = path.resolve(agWallet, htmlBasedir);
+  const agWalletHtmlUrl = await importMetaResolve(htmlBasePath, packageUrl);
+  const agWalletHtml = new URL(agWalletHtmlUrl).pathname;
 
   let hostport;
   await Promise.all(
